dbt/include/clickhouse/macros/materializations/view.sql

```diff
@@ -31,18 +32,30 @@
     {{ log('Creating new relation ' + target_relation.name )}}
     -- There is not existing relation, so we can just create
     {% call statement('main') -%}
-      {{ get_create_view_as_sql(target_relation, sql) }}
+      {% if matview %}
+        {{ clickhouse__create_matview_as(target_relation, sql) }}
+      {% else %}
+        {{ get_create_view_as_sql(target_relation, sql) }}
+      {% endif %}
     {%- endcall %}
   {% elif existing_relation.can_exchange %}
     -- We can do an atomic exchange, so no need for an intermediate
     {% call statement('main') -%}
-      {{ get_create_view_as_sql(backup_relation, sql) }}
+      {% if matview %}
+        {{ clickhouse__create_matview_as(target_relation, sql) }}
+      {% else %}
+        {{ get_create_view_as_sql(target_relation, sql) }}
```

Comment:
This line is what's breaking tests -- the original references `backup_relation` but the new line is `target_relations`